Mage.Sets/src/mage/cards/d/DihadaBinderOfWills.java

```diff
@@ -0,0 +1,211 @@
+package mage.cards.d;
```

Comment:
You need to add this card to the set. see: https://github.com/magefree/mage/blob/6d50e3d508766b62fbf315c3f8b12d2bd887ee02/Mage.Sets/src/mage/sets/DominariaUnited.java

Reply:
DominariaUnitedCommander, but yeah...